ci/pipelines/apps-test-ecr.yml

```diff
@@ -8,6 +8,14 @@ aws_config: &aws_config
   aws_ecr_registry_id: "((pay_aws_test_account_id))"
   aws_region: eu-west-1
 
+aws_staging_config: &aws_staging_config
+  aws_access_key_id: ((readonly_access_key_id))
+  aws_secret_access_key: ((readonly_secret_access_key))
+  aws_session_token: ((readonly_session_token))
+  aws_role_arn: arn:aws:iam::((pay_aws_staging_account_id)):role/concourse
+  aws_ecr_registry_id: "((pay_aws_staging_account_id))"
```

Comment:
Has this variable already been set in Concourse secrets?

Reply:
yep